website/docs/pipelines/promoting-applications.mdx

```diff
@@ -140,118 +142,170 @@ is upgraded in one of the pipeline's environments, a PR is created that updates
 The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files,
 how to authenticate with the repository and the Git provider API, and which file hosts the version definition for each environment.
 
-:::caution
+### Security
+
+:::danger
 
-Creating pull requests requires read and write access in your git repo.
-A compromised token could lead to changing the git repository and creation of legitimate looking changes/pull requests.
+Create pull requests requires write access to your git repo. In case your pull request secret is compromised (and it is
+suggested you assume it as a likely scenario more than unlikely), it could end up for example, in deleting all your
+production applications.
 
-Ensure you understand and adopt [security recommendations](./#security-recommendations) before using the feature.
+This section should not be read as best practices but as baseline to enable automated pull requests in lower environments.
 
+It targets platform admins before enabling the capability.
 :::
 
-### Supported Git Providers
-The following Git providers are currently support by this promotion strategy:
+#### Environments
 
-- [GitHub](https://github.com/)
-- [GitLab](https://gitlab.com/)
-- [BitBucket Server / DataCenter](https://www.atlassian.com/software/bitbucket/enterprise)
+1. **Use it in a lower environment when there are no attack vectors to production**: avoid using it if a compromised token could
+make changes in a lower environment (for example test or staging) that could be promoted to production being undetected
```

Comment:
extend with questions that you should ask as platform admin to make it a workable scenario - for example shared configuration repo (test, staging, prod)

we cannot provide checklist but guidance

ex solution leverage codeowners for extra eyes for production  

to review in the context of the shared responsibility model

Reply:
updated